types/gamedig/lib/GameResolver.d.ts

```diff
@@ -0,0 +1,45 @@
+import { Type } from 'gamedig';
```

Comment:
```suggestion
import { Type } from '.';
```

Reply:
is there any effective difference between the previous `import { Type } from './index';` and this or is omitting index just personal preference?

Reply:
really more of a miss on my part.
i should have read the "no-relative-import-in-test" thrown during testing more thoroughly, i foolishly assumed non relative imports were always preferred

Reply:
@rbuckton I'm assuming with this now being fixed with [e3f57d9](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62210/commits/e3f57d998a21c136d53486ddda368599004b3be3), that this is resolved and this PR can be merged.
If this is not the case please let me know. Otherwise I'm going to assume this to be approved as per your previous review, and merge this Sunday (~3pm EST)